bin/check_tron_jobs.py

```diff
@@ -64,7 +64,7 @@ def compute_check_result_for_job_runs(client, job, job_content):
     action_run_details = client.action_runs(action_run_id.url, num_lines=10)
 
     if is_job_stuck(job_content):
-        prefix = "STUCK"
+        prefix = "Job still running when next job is scheduled to run (stuck?)"
```

Comment:
`prefix` is supposed to be the first few words. make this "WARN" and come up with a different variable to represent this sentance